### DISCUSSION TITLE
Remove action row

### DISCUSSION THREAD

**Discussion by lian-hsc** [discord-jda/JDA#2327] [Questions and Help]
So basicly i send a message with tow buttons (allow, block).
If somebody clicks on one of them i edit to message to say, Action blocked/allowed by @Mention.
Then i also want to remove the buttons, but i can't figure out how. I tried
- `setActionRow()`
- `setActionRow(ItemComponent[0])` and
- without any `setActionRow`

None of them worked.
Thanks for your help in advance.
I used `v5.0.0-alpha.22`

**Comment by RedcodesDev** [comment]
To remove all components from a message you may use the `setComponents(Collections.emptyList())` method.

You can read more about it here:
https://ci.dv8tion.net/job/JDA5/javadoc/net/dv8tion/jda/api/utils/messages/MessageRequest.html#setComponents(java.util.Collection)
